src/BodyFunctionality/BodyPartGenerator.java

```diff
@@ -0,0 +1,382 @@
+package BodyFunctionality;
+
+import javafx.scene.paint.Color;
+
+import java.util.ArrayList;
+
+public class BodyPartGenerator
+{
+
+    enum BodyPartType{ NA, FINGER, HAND, ARM, WING, HEAD, MUZZLE, NAIL, HORN, TAIL, TONGUE, EYE, EAR, NOSE, MOUTH,
+                        NECK, STOMACH, HEART, MIND, SPECIAL}
+
+    enum Texture{ NA, SKIN, SCALED, LEATHER, HAIRY, NAIL, BONE, SPIKY, ROUGH, SPECIAL }
+
+    enum AnimalType{ NA, MIXED, HUMAN, DRAGON, UPRIGHTDRAGON, GUINEAPIG, OTHER }
+
+    enum Stance{NA, UPRIGHT, ONFEET, ONGROUND, OTHER}
+
+    enum Gender{NA, MALE, FEMALE, OTHER}
+
+    enum HandType{NA, HAND, FRONTCLAW, FRONTPAW, OTHER}
+
+    enum FootType{NA, FOOT, BACKCLAW, BACKPAW, GRASPINGFOOT, NONE, OTHER}
+
+    static BodyNail createNail(String name, AnimalType animalType)
```

Comment:
Do we still need these part generators?

Reply:
This depends on how the creatures are created.  That still needs to be decided.

Reply:
Created issue